docs/developer-guide/working-with-plugins.md

```diff
@@ -221,7 +221,7 @@ Add these keywords into your `package.json` file to make it easy for others to f
 
 ## Working with Custom Parsers
 
-If you want to use your own parser and provide additional capabilities for your rules, you can specify your own custom parser. If a `parseForESLint` method is exposed on the parser, this method will be used to parse the code. Otherwise, the `parse` method will be used. Both methods should take in the the source code as the first argument, and an optional configuration object as the second argument (provided as `parserOptions` in a config file). The `parse` method should simply return the AST. The `parseForESLint` method should return an object that contains the required property `ast` and an optional `services` property. `ast` should contain the AST. The `services` property can contain any parser-dependent services (such as type checkers for nodes). The value of the `services` property is available to rules as `context.parserServices`.
+If you want to use your own parser and provide additional capabilities for your rules, you can specify your own custom parser. If a `parseForESLint` method is exposed on the parser, this method will be used to parse the code. Otherwise, the `parse` method will be used. Both methods should take in the the source code as the first argument, and an optional configuration object as the second argument (provided as `parserOptions` in a config file). The `parse` method should simply return the AST. The `parseForESLint` method should return an object that contains the required property `ast` and an optional `services` property. `ast` should contain the AST. The `services` property can contain any parser-dependent services (such as type checkers for nodes). The value of the `services` property is available to rules as `context.parserServices`. You can either specify a path as parser or require your parser directly.
```

Comment:
To add onto this, I think it might be useful to give some additional context in this sentence. The previous paragraph discusses parsers in general without describing how they're used. Then the new sentence jumps directly into a detail about the available options for using a parser, without explaining how to use them in the first place. It might help to add a clause along the lines of "To use a parser, you can specify a module name or a parser object as the `parser` property in a config file."